pandas/tests/reshape/test_pivot.py

```diff
@@ -1288,6 +1288,39 @@ def test_pivot_number_of_levels_larger_than_int32(self):
             df.pivot_table(index='ind1', columns='ind2',
                            values='count', aggfunc='count')
 
+    @pytest.mark.parametrize('dropna', [
```

Comment:
you don't need to parameterize, we *already* have a dropna fixture, just remove these 2 lines and it will work